src/fides/api/ops/api/v1/endpoints/messaging_endpoints.py

```diff
@@ -231,6 +308,47 @@ def get_config_by_key(
         )
 
 
+@router.get(
+    MESSAGING_DEFAULT_BY_TYPE,
+    dependencies=[Security(verify_oauth_client, scopes=[MESSAGING_READ])],
+    response_model=MessagingConfigResponse,
+)
+def get_default_config_by_type(
+    service_type: MessagingServiceType, *, db: Session = Depends(deps.get_db)
+) -> MessagingConfig:
+    """
+    Retrieves default config for messaging service by type.
+    """
+    logger.info("Finding default messaging config of type '{}'", service_type)
+
+    messaging_config = MessagingConfig.get_by_type(db, service_type)
```

Comment:
I think this is confusing because when we add a "default", we use the `Default Messaging Config` pre-fix to denote that.  This `GET` will get any message config by type, not just the "default", right?

Reply:
Yes your comments helped clarify the meaning of "default" overall. Thanks!